### PR TITLE
Enhance scala code style check

### PR DESCRIPTION
**Pull request by gengliangwang**
Currently the code style check is weak. 
This PR copies most of the Scala style configs  from Apache Spark. And Fix all the errors on code style.

### CODE REVIEW

File: src/test/scala/com/databricks/spark/avro/AvroWriteBenchmark.scala [hunk 1]
```diff
@@ -19,14 +19,17 @@ package com.databricks.spark.avro
...
 import scala.util.Random
+
 import com.google.common.io.Files
 import org.apache.commons.io.FileUtils
+
 import org.apache.spark.sql._
 import org.apache.spark.sql.types._
 
-import scala.math.BigDecimal.RoundingMode
+// scalastyle:off println
```
Comment (gatorsmile) [inline src/test/scala/com/databricks/spark/avro/AvroWriteBenchmark.scala]: ?

Comment (gatorsmile) [reply src/test/scala/com/databricks/spark/avro/AvroWriteBenchmark.scala]:
```Scala
// scalastyle:off println
...
// scalastyle:on println
```

Comment (gengliangwang) [reply src/test/scala/com/databricks/spark/avro/AvroWriteBenchmark.scala]: This is for turning off `println` check for the rest of this scala file.

**Comment by gatorsmile** [comment]
LGTM

**Comment by gatorsmile** [comment]
ok to test

**Pull request merged**